app/controllers/article_controller.rb

```diff
@@ -19,6 +19,25 @@ def show
 		@article = Article.find(params[:id])
 	end
 
+	def edit
+		@article = Article.find(params[:id])
```

Comment:
@GangadharSwami 
indentations and spaces are very much important. 
always use 2 space indentation.

Reply:
https://github.com/rubocop-hq/ruby-style-guide#indentation

Reply:
ok